CRM/Upgrade/Incremental/php/FiveSixtyOne.php

```diff
@@ -21,6 +21,23 @@
  */
 class CRM_Upgrade_Incremental_php_FiveSixtyOne extends CRM_Upgrade_Incremental_Base {
 
+  public function setPreUpgradeMessage(&$preUpgradeMessage, $rev, $currentVer = NULL): void {
+    if ($rev === '5.61.alpha1' && CRM_Core_DAO::singleValueQuery('SELECT id FROM civicrm_contribution_recur LIMIT 1')) {
+      $documentationUrl = 'https://docs.civicrm.org/dev/en/latest/financial/recurring-contributions/';
+      $documentationAnchor = 'target="_blank" href="' . htmlentities($documentationUrl) . '"';
+      $extensionUrl = 'https://docs.civicrm.org/dev/en/latest/financial/recurring-contributions/';
+      $extensionAnchor = 'target="_blank" href="' . htmlentities($extensionUrl) . '"';
+
+      $preUpgradeMessage .= '<p>' .
+        ts('This release contains a change to the behaviour of recurring contributions under some edge-case circumstances.')
+        . ' ' . ts('In recent releases the amount and currency on the recurring contribution record changed when the amount of a contribution against it was changed, indicating a change in future intent.')
```

Comment:
Is the recur updating when *any* linked contribution is updated or just the latest?

Reply:
Currently any

Reply:
ok, please update the message to make it clear

Reply:
So you just want the word 'any' added in here "the amount of any contribution"

Reply:
`In recent releases` also what was the specific release - 5.49 - let's include it in the message to make it clear.

Reply:
Have posted this as a suggested code edit below

Reply:
thanks for checking the version - committed - I can rebase when we are happy